packages/core/types/stitches.d.ts

```diff
@@ -162,7 +168,7 @@ export default interface Stitches<
 			}
 		): StyledComponent.CssComponent<
 			StyledComponent.StyledComponentType<Composers>,
-			StyledComponent.StyledComponentProps<Composers>,
+			StyledComponent.StyledComponentProps<Composers, Media> & { css?: CSS },
```

Comment:
Should remove the css prop, due to new improved way.
```suggestion
			StyledComponent.StyledComponentProps<Composers, Media>,
```